### PR TITLE
Add fuction to calculate beta value in Finf

### PR DESCRIPTION
**Pull request by aitordiaz**

### CODE REVIEW

**Review by kasia-postdata — COMMENTED** [review]
Hi! Nice thanks,

We also need the function to  estimate best beta parameter value for the ICM (theorem 2)
Page 20,21 https://unedo365-my.sharepoint.com/personal/enrique_lsi_uned_es/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fenrique%5Flsi%5Funed%5Fes%2FDocuments%2FMicrosoft%20Teams%20Chat%20Files%2FCOMPOSITIONAL%5FDISTRIBUTIONAL%5F%5FACL%5FJOURNAL%5F2021%5F%2Epdf&parent=%2Fpersonal%2Fenrique%5Flsi%5Funed%5Fes%2FDocuments%2FMicrosoft%20Teams%20Chat%20Files

**Review by kasia-postdata — COMMENTED** [review]
Guys, but you haven't added F_inf operation to the "f" function (line 97)

**Comment by aitordiaz** [comment]
Oh, nice catch. We're going to upload the changes. Thank you!

File: src/utils.py [hunk 1]
```diff
@@ -166,3 +167,47 @@ def composition_function(f_name: str, embedding_list: List[np.array]):
...
+
+
+def calc_beta_icm(v1, v2):
+    r"""
+    Function to calculate an estimation of the optimal beta parameter:
+
+    .. math::
+        β = \frac{Avg(\|\vec{v}_1\|^2, \|\vec{v}_2\|^2)} {Avg(\|\vec{v}_1\|^2 + \|\vec{v}_2\|^2 - \left\langle \vec{v}_1, \vec{v}_2 \right\rangle)}
+
+    :param v1: composition vector 1
```
Comment (kasia-postdata) [inline src/utils.py]: hmmm why the input of the function is just 2 vectors if that should be an average over every pair in the dataset? The input should be list of tuples or sth
from the paper: "According to the studies in (?), given a collection of document pairs, the optimal parameter can be estimated as the ratio between the average sum of information quantities and the average joint information quantity of all instance pairs for which similarity must be estimated".

**Pull request merged**